client/params/config.go

```diff
@@ -23,6 +23,21 @@ var DefaultConfig = &Config{
 	CollationSizeLimit:      int64(math.Pow(float64(2), float64(20))),
 }
 
+// GetDefaultConfig returns a fresh DefaultConfig value following the factory pattern.
+func GetDefaultConfig() *Config {
```

Comment:
Can you remove the constant on 14 and fix references to it? 
I prefer a single source of DefaultConfig

Reply:
Done, I've added constants or functions for configuration params that are used in isolation, without needing a complete `Config`, let me know WDYT.